templates/lambda.template.yaml

```diff
@@ -398,8 +398,19 @@ Resources:
               - s3:GetObject
             Resource:
               - !Sub arn:aws:s3:::${QSS3BucketName}/${QSS3KeyPrefix}*
+          - Effect: Allow
+            Action:
+              - ssm:GetParameter
```

Comment:
We must allow dynamic pipeline maker lambda role access to parameters and codestar connections